docs/developers/analyzer-development.md

```diff
@@ -2,119 +2,342 @@
+https://github.com/google/timesketch/blob/master/data/features.yaml
+
+An example extraction entry looks like this:
 
 ```
-python3 analyzer_run.py
-usage: analyzer_run.py [-h] [--test_file PATH_TO_TEST_FILE] PATH_TO_ANALYZER NAME_OF_ANALYZER_CLASS
-analyzer_run.py: error: the following arguments are required: PATH_TO_ANALYZER, NAME_OF_ANALYZER_CLASS
+name:
+      query_string: 'my_secret_attribute:"look here"'
+      # Mandatory fields.
+      attribute: 'message'
+      Store_as: 'secret'
+      re: 'not a secret:([^\.]+).'
+      # Optional fields.
+      emojis: ['camera']
+      tags: ['secret-entry']
+      create_view: True
+      aggregate: False
 ```
 
-### create your sample data
+### Sigma Analyzer
+
+The Sigma analyzer uses sigma rules from the community
+[sigma project](https://github.com/SigmaHQ/sigma) and internally
+developed/adjusted rules to detect and tag events in a timeline. It allows for
+easy management of rules by modifying an existing rule or creating a new rule
+from scratch directly in the Timesketch web UI. Write a sigma rule instead of a
+new analyzer if you have a specific search query or an existing community rule
+and want to tag all those events.
+
+Check the
+[sigma analyzer docs](https://github.com/google/timesketch/blob/master/docs/guides/analyzers/sigma_analyzer.md)
+for more information.
+
+### Tagger Analyzer
 
-You can create your sample data either in CSV or JSONL with the same format
-that Timesketch can ingest. To learn more about that visit
-[CreateTimelineFromJSONorCSV](/guides/user/import-from-json-csv/)
+If your query requires a more advanced search query like dynamically tagging
+based on values derived from event attributes, you should make use of the tagger
+analyzer. This also allows you to tag all events based on a generic search query
+and regex searches. See the
+[tagger doc](https://github.com/google/timesketch/blob/master/docs/guides/analyzers/tagger.md)
+for more information.
 
-### use existing sample data
+TIP: If your detection idea can be implemented with a Sigma rule use that
+instead of the tagger analyzer!
 
-There is an event file shipped with Timesketch at:
+To use it, just add a new tagger entry in the `tags.yaml` file:
+https://github.com/google/timesketch/blob/master/data/tags.yaml
+
+An example tagger entry looks like this:
 
 ```
-timesketch/test_tools/test_events/sigma_events.jsonl
+test_tagger:
+    query_string: 'test'
+    tags: ['test-tag']
+    emojis: ['FISHING_POLE']
+    save_search: true
+    search_name: 'TEST the tag'
 ```
 
-### Running it with parameters
+## Build the analyzer
+
+Ok, let’s get started!
+
+### 1) Set Up your development environment
+
+Follow the instructions in
+[Timesketch development - Getting started](https://timesketch.org/developers/getting-started/)
+to get everything up and running.
+
+### 2) Generate the necessary analyzer templates
+
+Alright, you got your development environment up, great! The first thing we are
```

Comment:
IMHO; it is grat to use "you" etc in the doc, but that sounds a bit to "Umgangssprachlich"

Reply:
Point taken :D